src/routes/api/users.js

```diff
@@ -1,82 +1,11 @@
 import { Router } from 'express';
-import passport from 'passport';
+import { SendVerificationEmail } from '../../middlewares/mail';
+import { signUp, confirmEmailVerificaionToken } from '../../controllers/user';
 
 const router = Router();
 
-router.get('/user', (req, res, next) => {
-  User.findById(req.payload.id)
-    .then((user) => {
-      if (!user) {
-        return res.sendStatus(401);
-      }
-      return res.json({ user: user.toAuthJSON() });
-    })
-    .catch(next);
-});
+router.post('/email/test', SendVerificationEmail, signUp);
```

Comment:
Also why `email/test`? what is the purpose?

Reply:
Since i am implementing just the email verification feature, i felt having just a separate route would be fine. so when the signup feature is ready the `/email/test` middleware ( `SendVerificationEmail` ) would be passed as one of the middleware to signup